[IMP] Employees: Update certifications (#13063)
odoo/documentation: content/applications/hr/employees/certifications.rst

When jobs require specific knowledge, it is necessary to track employee certifications to ensure the
necessary knowledge and certifications are in place.

Certifications include classes, tests, professional seminars, and more. No restrictions are set in
terms of what type of certification records can be added in Odoo.

> .. important::

Author: It's ONLY needed if we are giving them an internal certification from a course we built. if not, you can just enter any certification. So Surverys is just a part of certifications - not the whole thing.

> .. important::

Author: It is- but I have the important part in the beginning - wouldn't that cover everything? 

And even though you don't need to use the surveys app for all certifications - if the app is not installed, you cannot see the report! So it is required for certifications, period (even though you don't need to USE surveys - but they are linked, which is why).

> .. important::

Author: I updated it to include that it is required to even access the section on the Employee form - which hs accurate!

this scenario, it is beneficial to view the certifications by expiration status.

To do so, navigate to :menuselection:`Employees app --> Reporting --> Certifications`. Next, click

> the :icon:`fa-caret-down` :guilabel:`(Toggle Search Panel)` icon in the search bar, then click

Reviewer: ```suggestion
the :icon:`fa-caret-down` :guilabel:`(dropdown)` icon in the search bar, then click
```

Author: I left it as this descripotion, because when you hover over the drop-down arrow, that's what it says- so I belive we are supposed to match the UI.

Reviewer 2: Lara is correct, @justinmallette !! We prioritize the text in the UI 
![image](https://github.com/user-attachments/assets/92d47633-ba2c-44bb-af91-90e6a7c75b5b)

Certifications`. Click :guilabel:`New`, and a blank certification form loads. Enter the following
information on the form:

> - :guilabel:`Title`: enter a short description for the certification in this field.

Reviewer: ```suggestion
- :guilabel:`Title`: Enter a short description for the certification in this field.
```

Reviewer: full sentence

> - :guilabel:`Employee`: using the drop-down menu, select the employee who received the

Reviewer: ```suggestion
- :guilabel:`Employee`: Using the drop-down menu, select the employee who received the
```

  certification.

> - :guilabel:`Type`: using the drop-down menu, select the type of certification received. This field

Reviewer: ```suggestion
- :guilabel:`Type`: Using the drop-down menu, select the type of certification received. This field
```

>     section of the *Resume* tab on the :doc:`employee form <new_employee>`.

Reviewer 2: nitpick on line 79: how about using brackets to indicate user input? Create [type]

  - :guilabel:`Education`: Select this option to have the certification appear in the *Education*
    section of the *Resume* tab on the :doc:`employee form <new_employee>`.
  - :guilabel:`Social Media`: Select this option to have the certification appear in the *Social

>     Media* section of the *Resume* tab on the :doc:`employee form <new_employee>`.

Reviewer 2: slight nitpick, but what does adding the social media type do, exactly? how is it useful for the new employee form? Let's add a sentence after this one explaining why this is helpful for a business or employee logs (:

Author: I added an example for all of the entries to specify what is typically added here. Good suggestion!

  - :guilabel:`Internal Certification`: Select this option to have the certification appear in the
    *Internal Certification* section of the *Resume* tab on the :doc:`employee form <new_employee>`.
  - :guilabel:`Completed Internal Training`: Select this option to have the certification appear in
    *Completed Internal Training* section of the *Resume* tab on the :doc:`employee form

>     <new_employee>`.

Reviewer 2: was wondering why this was removed, since i see it in runbot

![image](https://github.com/user-attachments/assets/8ca6e289-9cf9-4864-8b35-801b9b2d09b4)

I know you thoroughly test, so maybe something needs to be enabled in order for this field to appear?

Author: This did not appear in my database when I originally created it - but now it does! Adding it now - I thought this was a custom addition in Runbot!

    section of the *Resume* tab on the :doc:`employee form <new_employee>`.
  - :guilabel:`Education`: Select this option to have the certification appear in the *Education*
    section of the *Resume* tab on the :doc:`employee form <new_employee>`.
  - :guilabel:`Social Media`: Select this option to have the certification appear in the *Social
    Media* section of the *Resume* tab on the :doc:`employee form <new_employee>`.
  - :guilabel:`Internal Certification`: Select this option to have the certification appear in the
    *Internal Certification* section of the *Resume* tab on the :doc:`employee form <new_employee>`.

- :guilabel:`Display Type`: Select the visibility of the certification in this field. The default
  options are:

>   - :guilabel:`Classic`: select this option to have the certification appear in the *Resume* section

Reviewer: ```suggestion
  - :guilabel:`Classic`: Select this option to have the certification appear in the *Resume* section
```

    of the employee form, and **not** appear on the *Employee Certifications* report.

>   - :guilabel:`Course`: select this option to have the certification appear in the *Resume* section

Reviewer: ```suggestion
  - :guilabel:`Course`: Select this option to have the certification appear in the *Resume* section
```

    of the employee form, and **not** appear on the *Employee Certifications* report. Once this
    option is selected, a :guilabel:`Course` field appears beneath the :guilabel:`Display Type`
    field. Using the drop-down menu, select the corresponding **eLearning** course the employee
    took. This field **only** appears if the **eLearning** app is installed.
  - :guilabel:`Certification`: select this option to have the certification appear in the *Resume*
    section of the employee form, **and** appear on the *Employee Certifications* report. Once this
    is selected, a :guilabel:`Certification` field appears beneath the :guilabel:`Display Type`
    field. Using the drop-down menu, select the corresponding **Surveys** app certification the
    employee took. This field **only** appears if the **Surveys** app is installed.

> - :guilabel:`Description`: enter a description for the certification in this field.

Reviewer: ```suggestion
- :guilabel:`Description`: Enter a description for the certification in this field.
```

> - :guilabel:`Duration`: click into the first field, and a calendar pop-over window appears. Click on

Reviewer: ```suggestion
- :guilabel:`Duration`: Click into the first field, and a calendar pop-over window appears. Click on
```

  - :guilabel:`Course`: Select this option to have the certification appear in the *Resume* section
    of the employee form, and **not** appear on the *Employee Certifications* report. Once this
    option is selected, a :guilabel:`Course` field appears beneath the :guilabel:`Display Type`
    field. Using the drop-down menu, select the course the employee took. The course is created in
    the **Surveys** app.

>   - :guilabel:`Certification`: Select this option to have the certification appear in the *Resume*

Reviewer 2: I noticed that in the runbot, the order is Classic, Certification, then Course. Let's match the UI in order, too (: 

![image](https://github.com/user-attachments/assets/46cb806b-4779-4e3a-b620-d24e82ceb6b2)

>     field. Using the drop-down menu, select the corresponding **eLearning** course the employee

Author: Ahh, OK - I was thinking this was an **eLearning** APP course - so that's why I made it bold. But I understand your logic here.
